### ISSUE TITLE
High WER with mixed precision training

### ISSUE THREAD

**Issue by k18shruti** [SeanNaren/deepspeech.pytorch#447]
I was able to get a smaller validation WER in the floating-point mode when applied to the LibriSpeech dataset. But when I gave the option of --mixed-precision, I don't see any decrease in the WER. It just stays at 100. Here is the command that I used:
python train.py --train-manifest train_manifest.csv --val-manifest val_manifest.csv --cuda --epochs 20 --model-path model_tmp/deepspeech_final.pth --save-folder model_tmp --batch-size 64 --mixed-precision --dynamic-loss-scale

I am running it on V100 card.
The rest of the parameters are the same as for fp32. Could you let me know if there are any other options that I can try to modify?

**Comment by SeanNaren** [comment]
Try using a static loss scale with a loss scale of `1`. it seems dynamic loss scale may not help with training the current deepspeech model

**Comment by SeanNaren** [comment]
I'll close this for now. Just for additional context, currently using warp-ctc the gradients cannot be scaled hence dynamic-loss-scale does nothing.
